helm/values.yaml

```diff
@@ -9,7 +9,7 @@ images:
   ucentralgw:
     repository: tip-tip-wlan-cloud-ucentral.jfrog.io/ucentralgw
     tag: master
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
```

Comment:
Why do we change this to `IfNotPresent`?

Reply:
Partly agree but make sure to communicate this change with Jaspreet because he probably expects that deleting a pod will upgrade it to the latest version.

Reply:
@dunaev-opsfleet I don't think these two things are connected - if you want to use another tag which is a commit -specific, you still should user pullPolicy Always, it won't do any harm and also allow to use master tag in other cases in which you do want to re-pull in case the pod restarts.

Reply:
OK, got it, will change it back